src/components/ScenarioParameters/ScenarioParameters.js

```diff
@@ -239,7 +240,7 @@ const ScenarioParameters = ({
               {t('genericcomponent.text.scenario.parameters.title', 'Scenario parameters')}
             </Typography>
           </Grid>
-          <PermissionsGate requiredPermissions={[PERMISSIONS.canEditOrLaunchScenario]}>
+          <PermissionsGate authorizedPermissions={[PERMISSIONS.canEditOrLaunchScenario]}>
```

Comment:
IMO, `requiredPermissions` sounds better than `authorizedPermissions` in this scope :thinking:

Reply:
:man_shrugging: dunno :p